routes/api.php

```diff
@@ -18,9 +19,8 @@
     return $request->user();
 });
 
-
-Route::prefix("V1")->group(function () {
+Route::prefix("v1")->group(function () {
     Route::apiResource('achievements', App\Http\Controllers\Api\V1\AchievementController::class);
+    Route::apiResource('challengers', ChallengerController::class)->only('show');
```

Comment:
Nice!  I didn't think of allowing just finished endpoints, I'll do it In my next pr.